src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -99,7 +101,7 @@ similar to this:
 Lint does not report on APIs that are not implemented by Firefox or Firefox for Android.
 :::
 
-When setting `strict_min_version` in [`browser_specific_settings`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings), unless you’re targeting a specific version of Firefox, choose the most recent version of Firefox you expect your extension to be compatible with. Due to the different APIs and form factors on Android (compared to desktop Firefox), set `gecko_android` after explicitly verifying compatibility. This field enables a compatibility range that is distinct from Firefox for desktop.
+When setting `strict_min_version` values in `browser_specific_settings`, unless you’re targeting a specific version of Firefox, choose the most recent version of Firefox you expect your extension to be compatible with. Due to the different APIs and form factors on Android (compared to desktop Firefox), set `gecko_android` after explicitly verifying compatibility. This field enables a compatibility range that is distinct from Firefox for desktop.
```

Comment:
"`suggestion
When setting `strict_min_version` values in `browser_specific_settings`, unless you're targeting a specific version of Firefox, choose the most recent version of Firefox you expect your extension to be compatible with. Due to Android's different APIs and form factors (compared to desktop Firefox), set `gecko_android` after explicitly verifying compatibility. This sub-key enables a compatibility range that is distinct from Firefox for desktop.
```